File: MP1/Temperature.cpp
#include <iostream>
using namespace std;

int main(){
    float f, c;

    cout << "Enter temperature in Fahrenheit:";
    cin >> f;

    c = ((f - 32) * 5)/9;
    cout << "Celsius: " << setprecision(2) << c << endl;

    return 0;
}

File: MP1/Temperature.java
import java.util.Scanner;


public class Temperature{
    public static void main(String args[]){
        float f;
        
        Scanner input = new Scanner(System.in);
        System.out.print("Enter temperature in Fahrenheit:");
        f = input.nextFloat();

        float c = ((f - 32) * 5)/9;
        System.out.println("Celsius :" + c);
        input.close();
    }
}